module/VuFind/src/VuFind/View/Helper/Root/RecordDataFormatterFactory.php

```diff
@@ -333,6 +333,10 @@ public function getDefaultCoreSpecs()
                 'itemSuffix' => '</span>',
             ]
         );
+        $spec->setLine(
```

Comment:
I wonder if it would be safer to make this an example in RecordDataFormatter.ini instead of turning it on by default. Since 866 is technically part of MARC holdings, it's not really a standards-compliant thing to do by default. I realize this is a common use case because of merging holdings records into bib records, but I imagine turning it on automatically might have unwanted/unexpected side effects for many people. Making it easy to activate while leaving it off to begin with seems like the safest path forward (though I'm open to discussing further if you or others feel differently).

Reply:
thanks @demiankatz . Yes, that makes perfect sense. In my latest follow-up I've shifted this over to be an example in the RecordDataFormatter.ini file. As I am not very familiar with that file I am not sure if I have added it correctly. If you wouldn;t mind checking?